src/components/InputField/InputField.stories.mdx

```diff
@@ -42,6 +42,10 @@ export const inputWarningArgTypes = {
     },
 };
 
+export const disabledPropsInControl = {
```

Comment:
**thought:** we can rename this variable to `argTypes` 
also, we don't need to export it, right ?

Reply:
We need to export it otherwise mdx considers it as plain text.